### ISSUE TITLE
Competitor tutorial link for newcomers

### ISSUE THREAD

**Issue by denthebro** [thewca/worldcubeassociation.org#5707]
Can/should we link to [this competitor tutorial](https://www.youtube.com/watch?v=dPL3eV-A0ww) when a newcomer creates a WCA account?
Something among the lines of "First time competing? Check out this video to familiarize yourself with WCA competitions!"

**Comment by EdHollingdale** [comment]
A significant proportion of new competitors do not speak English, and/or live in regions where the runner system is not used, so this tutorial may not be appropriate. I send this tutorial out to competitors in my region when I am organising, but I don't think the WCA should be sending it to all competitors. 

Plus competition practices may be very different in the Covid-safe era, so showing a tutorial like this could add to confusion for first-time competitors.

**Comment by denthebro** [comment]
I agree with the Covid thing, maybe right now is not good timing, but once things get back to normal, I think it would be a lot more useful for newcomers than nothing at all. There are subtitles for a number of languages on there too.

**Comment by denthebro** [comment]
Upon looking at this again, I believe that we should instead link to the competitor tutorial made by the WQAC, that is on the website right now (but will likely be updated in the future)
